tests/TestClassSDKWrapper.php

```diff
@@ -134,24 +134,14 @@ public function testAddRequestByApiAction(): APIClientActionGeneric
 
 	public function testGetCache()
 	{
+		/** @var DBCache $pDBCaches */
 		$pDBCaches = $this->_pSDKWrapper->getCache();
+		$pDBCaches->clearAll();
 		$this->assertCount(1, $pDBCaches);
 		$this->assertInstanceOf(DBCache::class, $pDBCaches[0]);
 		$this->assertInstanceOf(onOfficeSDKCache::class, $pDBCaches[0]);
 	}
 
-	/**
-	 *
-	 */
-
-	public function testWriteCache()
-	{
```

Comment:
Has this been broken?

Reply:
@jayay Yes, I added it. It worked on my PC, but not on Github.